File: gif_lib/src/main/java/com/vam/giflib/NativeLib.java
package com.vam.giflib;

public class NativeLib {

    // Used to load the 'giflib' library on application startup.
    static {
        System.loadLibrary("giflib");
    }

    /**
     * A native method that is implemented by the 'giflib' native library,
     * which is packaged with this application.
     */
    public native String stringFromJNI();
}

File: gif_lib2/src/main/java/com/example/gif_lib2/NativeLib.kt
package com.example.gif_lib2

class NativeLib {

    /**
     * A native method that is implemented by the 'gif_lib2' native library,
     * which is packaged with this application.
     */
    external fun stringFromJNI(): String

    companion object {
        // Used to load the 'gif_lib2' library on application startup.
        init {
            System.loadLibrary("gif_lib2")
        }
    }
}